pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -518,7 +518,7 @@ func (m *cgroupManagerImpl) Pids(name CgroupName) []int {
 		// Walk through the pod cgroup directory to check if
 		// container cgroups haven't been GCed yet. Get attached processes to
 		// all such unwanted containers under the pod cgroup
-		if err = filepath.Walk(dir, visitor); err != nil {
+		if err = filepath.WalkDir(dir, visitor); err != nil {
```

Comment:
As if the parameter named fn for filepath.Walk is defined as  type WalkFunc func(path string, info fs.FileInfo, err error) error and the one for filepath.WalkDir is defined as type WalkDirFunc func(path string, d DirEntry, err error) error. So we can not  use WalkDirFunc instead of WalkFunc without change defination of  fn.